src/app/presentation/components/home-components/about/about.component.html

```diff
@@ -46,9 +46,9 @@ <h3 class="feature-title">About team</h3>
             <mat-grid-tile>
               <div class="about-text text-right justify-start">
                 <h3 class="feature-title">About our product</h3>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore.</p>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                <p>In my country there is a problem with small buisnes, because of problems with managment. Company owners doesnt want to spending money for managers, justifying this by the fact that their company doesnt so big, and all employers can manage their job on their own. And as a rule, thats strategy leads to the chaos, and total misunderstanding between owners and employees.</p>
+                <p>Our goal is to have all work live in Provis, while simplify the application as much as possible, with retaining all the functionality - thereby making people more productive and giving back at least 15% of time to dedicate to other things. </p>
+                <p>With our system you can comfortable: delegating, commenting tasks, follow up on assignments, keep track of deadlines, observe the efficiency of employees, better understand how much time a specific task takes, and based on this knowledge better plan your business in the future. And this is only part of the possibilities that Provis can propose you.</p>
```

Comment:
With our system you can comfortably: delegate, comment tasks, follow up on assignments, keep track of deadlines.
Also it allows you to observe the efficiency of employees, better understand how much time a specific task takes. Based on this knowledge you will plan your business better in the future. This is the least that Provis can propose you.